packages/terra-clinical-data-grid/src/proptypes/cellDataShape.js

```diff
@@ -0,0 +1,11 @@
+import PropTypes from 'prop-types';
+
+const cellDataShape = PropTypes.shape({
```

Comment:
These feel explanatory, but we should prob add in-line descriptions of these props

Reply:
With that, should this be named `cellDataPropTypes`?

Reply:
Good point, I can copy the descriptions from the docs and put them inline here.

Regarding the renaming, `cellDataPropTypes` would indicate (to me) that there's a CellData component somewhere that has prop types vs. `cellDataShape` which is a more literal descriptions of what the object is. But I won't die on a semantics hill so let me know if you feel real strongly one way or another.

Reply:
No strong opinion!